ceilometer_publisher_zeromq/queue.py

```diff
@@ -0,0 +1,55 @@
+from ceilometer import publisher
+from ceilometer.openstack.common.gettextutils import _
+from ceilometer.openstack.common import log
+from oslo.config import cfg
+import json
+import zmq
+
+LOG = log.getLogger(__name__)
+
+OPTS = [
+    cfg.StrOpt('publisher_zeromq_host',
+               default='127.0.0.1'),
```

Comment:
I don't think we can. ZMQ doesn't support the use of localhost, addresses are tied to transport and for tcp they much be numeric.